src/charm_state.py

```diff
@@ -573,6 +578,7 @@ def from_charm(cls, charm: CharmBase) -> "CharmConfig":
                 APROXY_REDIRECT_PORTS_CONFIG_NAME
             ),
             custom_pre_job_script=custom_pre_job_script,
+            runner_manager_log_level=runner_manager_log_level,  # type: ignore
```

Comment:
I believe the type ignore is because of the literal class vs string? Would there be a way to cast the string into the literals?